CHANGELOG.md

```diff
@@ -6,26 +6,37 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.10.0 (2023-02-16)
+
+This release contains new features and bug fixes since the 2.9.3 release.
+We deem it moderate priority for upgrading.
+
 **Features**
+* #4874 Allow joins in continuous aggregates
+* #4926 Refactor INSERT into compressed chunks
 * #5241 Allow RETURNING clause when inserting into compressed chunks
-* #5245 Mange life-cycle of connections via memory contexts
+* #5243 Enable real-time aggregation for continuous aggregates with joins
```

Comment:
Why another CHANGELOG entry if it is part of "Allow joins in continuous aggregates"?? Perhaps we should mix both together in the same entry

Reply:
I will definitely propose to do so on the release notes and the announcement; your call on the changelog (I am not sure if we want to list every single PR or not)

Reply:
In this case IMHO just the first entry is enough since we reduce the scope of the CAggs Joins and have time to add more capabilities on it before release.